cedar-policy/src/api.rs

```diff
@@ -170,6 +170,15 @@ impl Entities {
         Self(self.0.partial())
     }
 
+    /// Attempt to eagerly compute the values of attributes for all entities in the slice.
+    /// This can fail if evaluation of the [`RestrictedExpression`] fails.
+    /// In a future major version, we will likely make this function automatically called via the constructor.
```

Comment:
why does that change have to wait for next major version?

Reply:
Because it introduces a new way creation of an entities object can fail, and the `EntityError` enum isn't `non_exhaustive`